scripts/hooks/generate-docu.sh

```diff
@@ -6,7 +6,7 @@
 # ./scripts/hooks/generate-docu.sh
 
 function check_and_generate_docu() {
-  staged_changes=$(git diff --cached --name-only --diff-filter=ACM | grep -e "core/src/core-api" -e "client/src" | wc -l)
+  staged_changes=$(git diff --cached --name-only --diff-filter=ACM | grep -e "core/src/core-api" -e "client/src" -e "container/src"| wc -l)
```

Comment:
Sometimes it also happens that we only changes typings in container/typings only without touching code in container/src. This line doesnt capture it, so make sure to put it in as well.